packages/wonder-blocks-banner/src/components/__docs__/banner.stories.js

```diff
@@ -0,0 +1,355 @@
+                <Banner text="kind: info" kind="info" layout="floating" />
+            </View>
+            <View style={styles.spacing}>
+                <Banner text="kind: success" kind="success" layout="floating" />
+            </View>
+            <View style={styles.spacing}>
+                <Banner text="kind: warning" kind="warning" layout="floating" />
+            </View>
+            <View style={styles.spacing}>
+                <Banner
+                    text="kind: critical"
+                    kind="critical"
+                    layout="floating"
+                />
+            </View>
+        </View>
+    </View>
+);
+
+Variants.parameters = {
+    docs: {
+        storyDescription: `Banners come with two layouts: full-width (default)
+            and floating. Full-width layout gives the banner square edges,
+            and floating layout gives the banner rounded edges. Banners
+            also come in four different kinds: info (default), success,
+            warning, and critical.`,
+    },
+};
+
+export const DarkBackground: StoryComponentType = () => (
+    <View style={styles.darkBackground}>
+        <View style={styles.spacing}>
+            <Banner text="kind: info" kind="info" layout="floating" />
+        </View>
+        <View style={styles.spacing}>
+            <Banner text="kind: success" kind="success" layout="floating" />
+        </View>
+        <View style={styles.spacing}>
+            <Banner text="kind: warning" kind="warning" layout="floating" />
+        </View>
+        <View style={styles.spacing}>
+            <Banner text="kind: critical" kind="critical" layout="floating" />
+        </View>
+    </View>
+);
+
+DarkBackground.parameters = {
+    docs: {
+        storyDescription: "This is how banners look on a dark background.",
+    },
+};
+
+export const WithButtons: StoryComponentType = () => (
+    <Banner
+        text="This is a banner with buttons."
+        actions={[
+            {title: "Button 1", onClick: () => {}},
+            {title: "Button 2", onClick: () => {}},
+        ]}
+    />
+);
+
+WithButtons.parameters = {
+    docs: {
+        storyDescription: `This is a banner with buttons. An action, passed
+            into the \`actions\` prop, becomes a button when it has an
+            \`onClick\` value and does not have an \`href\` value.`,
+    },
+};
+
+export const WithLinks: StoryComponentType = () => (
+    <Banner
+        text="This is a banner with links."
+        actions={[
+            {title: "Link 1", href: "/"},
+            {title: "Link 2", href: "/", onClick: () => {}},
+        ]}
+    />
+);
+
+WithLinks.parameters = {
+    docs: {
+        storyDescription: `This is a banner with links. An action, passed
+            into the \`actions\` prop, becomes a link when it has an
+            \`href\` value. It can also have an \`onClick\` value, but it
+            will be a link regardless if it navigates to a URL via \`href\`.`,
```

Comment:
This makes me think about my previous comment of `href` vs `onClick` 🤔.

Reply:
I think this comment is still valid, b/c the semantics of the action will remain the same (`<a>` for this case).